api-reference/echo-operations/account-management.md

```diff
@@ -7,39 +7,28 @@ This operation is used to create an account
 ```cpp
 struct account_create_operation
 {
-  struct ext
-  {
-      optional< void_t >            null_ext;
-      optional< special_authority > owner_special_authority;
-      optional< special_authority > active_special_authority;
-      optional< buyback_account_options > buyback_options;
-  };
-
-  struct fee_parameters_type
-  {
-      uint64_t basic_fee      = 5*ECHO_BLOCKCHAIN_PRECISION; ///< the cost to register the cheapest non-free account
-      uint64_t premium_fee    = 2000*ECHO_BLOCKCHAIN_PRECISION; ///< the cost to register the cheapest non-free account
-      uint32_t price_per_kbyte = ECHO_BLOCKCHAIN_PRECISION;
-  };
-
-  asset           fee;
-  /// This account pays the fee. Must be a lifetime member.
```

Comment:
`/// This account pays the fee. Must be a lifetime member.`
Delete row - any account can create another account.

Reply:
fixed